static/js/cm-monokai-theme.ts

```diff
@@ -13,10 +13,10 @@ const strawberry = "#ff6188",
     stone = "#7d8799", 
     malibu = "#61afef",
     violet = "#ae81ff",
-    background = "#272822",
+    background = "#0F1E33",
     highlightBackground = "#202020",
     selection = "#90cdf463",
-    gutterBackground = "#2F3129",
+    gutterBackground = "#0F1E33",
```

Comment:
In my very personal and very biased opinion, I liked it more the way it was 😆 I can give an objective argument: I think the contrast between the keywords and the background looked better with the original theme, after all the theme wasn't designed by us, but rather by other people and we are just using it.